packages/mwp-app-route-plugin/src/index.js

```diff
@@ -1,18 +1,38 @@
 // @flow
+import LaunchDarkly from 'ldclient-node';
 import getRoute from './route';
 
+const LAUNCH_DARKLY_SDK_KEY = 'sdk-86b4c7a9-a450-4527-a572-c80a603a200f';
```

Comment:
why not make this a something travis fills in?

Reply:
mainly because that pushes responsibility to the apps to handle the key rather than it being something that Just Works without the apps having to do anything in particular - I've set it up so that in theory you pass in a unique `ldkey` (although the source of that option would need to be set up with a code update), but my understanding is that we'll be able to consolidate all LaunchDarkly flags under one 'account' for the time being.

Reply:
True, just more was trying to avoid keys in our code :)

Reply:
yeah I should probably follow up to set this differently